### PR TITLE
Insert Statistics Summary of scalardl-test  to log file

### PR DESCRIPTION
**Pull request by scalar-boney**
Insert `Statistics Summary` of scalardl-test to log file

### CODE REVIEW

File: src/main/java/com/scalar/kelpie/modules/PostProcessor.java [hunk 1]
```diff
@@ -37,6 +37,6 @@ protected void getSummary() {
       return;
     }
 
-    logInfo(stats.getSummary());
+    stats.getSummary();
```
Comment (feeblefakie) [inline src/main/java/com/scalar/kelpie/modules/PostProcessor.java:40]: Why do we need this change?
It seems already logging the summary with info.

Comment (scalar-boney) [reply src/main/java/com/scalar/kelpie/modules/PostProcessor.java:40]: @feeblefakie san,

Scalardl kelpie-test overrides this method.

https://github.com/scalar-labs/kelpie-test/blob/master/scalardl-test/src/main/java/scalardl/transfer/TransferReporter.java#L13

Comment (feeblefakie) [reply src/main/java/com/scalar/kelpie/modules/PostProcessor.java:40]: Isn't it a bug of TransferReporter?
It doesn't make sense to me to update the base.

**Review by feeblefakie — CHANGES_REQUESTED** [review]

File: src/main/java/com/scalar/kelpie/modules/PostProcessor.java (hunk 1 above)
Comment (feeblefakie) [reply src/main/java/com/scalar/kelpie/modules/PostProcessor.java:40]: Making TransferReporter not override `execute` causes any issues?

**Comment by scalar-boney** [comment]
PR closing because we found some alternatives in the script.

**Pull request closed without merging**